.swiftformat

```diff
@@ -9,8 +9,10 @@
 --ifdef no-indent
 --importgrouping testable-bottom
 --maxwidth 200
+--wraparguments after-first
 --wrapparameters after-first
```

Comment:
[discussion]
In [my original PR](https://github.com/nerdishbynature/octokit.swift/pull/117/files?w=1#diff-716e9196614e4df70d21cfe5515e79b2695a42931ca929e2654c88e27c081245R12), this value was `before-first` to wrap all parameters.
Is it particular with setting `after-first` ?

Reply:
Nope, maybe I copied it incorrectly. I'm all for your change!